core/pfe-core/ssr-shims.ts

```diff
@@ -52,7 +28,27 @@ globalThis.getComputedStyle ??= function() {
       return '';
     },
   };
-}
+};
 
-;
+// @ts-expect-error: opt in to event support in ssr
+globalThis.litSsrCallConnectedCallback = true;
```

Comment:
should we really be making this decision for our users? or should e.g. rhds' ssr regime choose whether or not to enable this?

Reply:
Making this decision for users means that yes they would have to refactor their elements, any time ssr-shims.js is loaded. i.e. if they use the `renderGlobal` function from `@patternfly/pfe-tools/ssr/global.js`. adding event listeners would not be a problem, because `EventTarget` is supported in node, but DOM access would throw. see element changes in this PR.

`isServer` is a boolean exported from a module in the lit package with import and node conditions, so it exports `true` when the node condition obtains and `false` otherwise. I don't see how wrapping that in a method would help.

Reply:
This is a major change for core, and a patch for elements.